velox/exec/PartitionedOutputBufferManager.h

```diff
@@ -265,7 +284,8 @@ class PartitionedOutputBufferManager {
  private:
   // Retrieves the set of buffers for a query.
```

Comment:
Let's clarify that this method throws is buffer for the specified task does not exist.

Reply:
I still think we should clarify that this method throws when task is not found.

```
// Retrieves the set of buffers for the specified task. Throws buffers-no-found exception if not found.
```